### ISSUE TITLE
When recording to a new track, the track area doesn't automatically scroll up to show the new track in full.

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#6340]
### Steps to reproduce

1. Have enough tracks to occupy the entire window's height
2. Shift + R to record to a new track
3. Observe

### Expected behavior

<img width="500" alt="image" src="https://github.com/audacity/audacity/assets/70063970/620227c2-f353-4904-b506-3bfaa7a9c0ad">

### Actual behavior

The window doesn't scroll up to put the new track entirely in the window:

<img width="500" alt="image" src="https://github.com/audacity/audacity/assets/70063970/f2f38f88-cc9e-41e0-89d2-e3de28e2c4dd">

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows, macOS, Linux

**Comment by Paul-Licameli** [comment]
Bisected exactly to 2418ec4286e075a9342f626ae35ba92779236dc7

**Comment by LWinterberg** [comment]
> Should it be in 3.5.2?

We'll have a pretty rapid 3.6 coming up, so there may not be a need for a 3.5.2. We'll discuss it on friday.

**Comment by petersampsonaudacity** [comment]
But the workarounds are pretty simple

a) use the vertical scroll bar to bring the track in progrees of being recorded into full view, or

b) just use the keyboard down-arrow

**Comment by dozzzzer** [comment]
I agree, but some might say this is a deal breaker for VO professionals who have their workflow perfectly tuned.

**Comment by Paul-Licameli** [comment]
No 3.5 patch for this?  Only a merge to master?

**Comment by dozzzzer** [comment]
> No 3.5 patch for this? Only a merge to master?

Correct, it will be shipped with 3.6
